tests/www/views/test_views_dagrun.py

```diff
@@ -290,3 +290,79 @@ def test_dag_runs_queue_new_tasks_action(session, admin_client, completed_dag_ru
     check_content_in_response("runme_2", resp)
     check_content_not_in_response("runme_1", resp)
     assert resp.status_code == 200
+
+
+@pytest.fixture
+def dag_run_with_all_done_task(session):
+    """Creates a DAG run for example_bash_decorator with tasks in various states and an ALL_DONE task not yet run."""
+    dag = DagBag().get_dag("example_bash_decorator")
+
+    # Re-sync the DAG to the DB
+    dag.sync_to_db()
+
+    execution_date = timezone.datetime(2016, 1, 9)
+    triggered_by_kwargs = {"triggered_by": DagRunTriggeredByType.TEST} if AIRFLOW_V_3_0_PLUS else {}
```

Comment:
Test are failing because of `AIRFLOW_V_3_0_PLUS` I think

Reply:
Yup, `AIRFLOW_V_3_0_PLUS` doesn't exist in this branch.

Reply:
I changed the tests for only < 3.0 case.